backend/app/exporters/models/marc21.rb

```diff
@@ -498,8 +498,9 @@ def handle_notes(notes)
                   end
 
       unless marc_args.nil?
+        is_561 = note['type'] == "custodhist" 
```

Comment:
@avatar382 I don't think this is what was intended from the spreadsheet.

Reply:
Yeah, looking at this now, I'm not happy with this logic. Thankfully, I revisited this with the more general "include unpublished" feature in branch ANW-376 which removes this code: 
https://github.com/avatar382/archivesspace/commit/0adc2f8e8394a51b1c2d88fcadc58112bef4d014

I'm thinking it would be best to just roll this change back in this branch, since it's superseded in the other.

Reply:
That sounds good to me!